Source/WebKit/Platform/IPC/ArgumentCoders.cpp

```diff
@@ -44,15 +44,13 @@ void ArgumentCoder<String>::encode(Encoder& encoder, const String& string)
         return;
     }
 
-    unsigned length = string.length();
     bool is8Bit = string.is8Bit();
-
-    encoder << length << is8Bit;
+    encoder << string.length() << is8Bit;
```

Comment:
I'm surprised that `encodeSpan` does not encode the length.

Reply:
It currently doesn't for some reason. However, it does seem like it probably should.